compositions/upbound-aws-provider/iam-policy/definition.yaml

```diff
@@ -56,7 +56,7 @@ spec:
                   - providerConfigName
                   type: object
               required:
-              - resourceArn
```

Comment:
This is to allow to use IAMPolicy and just create Policy as you see here https://github.com/csantanapr/crossplane-on-eks/blob/dynatrace-log-forwarder/compositions/upbound-aws-provider/iam-policy/cloudwatch-metrics-write.yaml
There is no need to pass resourceArn it will not be use there is no Policy attachment

Reply:
I was creating the Policy by it self as managed resource in the composition without IAMPolicy, you think that's better?

Reply:
IAMPolicy is there so that IAM policy defined in compositions can be reused. For example, read access to a bucket is a very common pattern so we should define it once and reuse it. Very similar to how SAM policy templates work. If you think the access pattern is not common, it's not necessary to define a new composition imo.

Reply:
my bad there is a policy attachment, I added as required field and used "*" as input in the compostion